docs/source/geo_def.rst

```diff
@@ -63,22 +62,25 @@ Creating an area definition:
 
 pyresample.utils
 ****************
-The utils module of pyresample has convenience functions for constructing
-area defintions. The function **get_area_def** can construct an area definition
-based on area extent and a proj4-string or a list of proj4 arguments.
+
+The :mod:`utils <utils>` module of Pyresample
+has convenience functions for constructing area definitions. The function
+:mod:`get_area_def <utils.get_area_def>` can construct an
+:mod:`AreaDefinition <geometry.AreaDefinition>` object based on
+area_extent and a proj4-string/dict or a list of proj4 arguments.
```

Comment:
What does "list of proj4 arguments" mean here?

Reply:
I didn't touch the wording of that, but I can reword it! It means that instead of providing '+a=6371228.0 +units=m +lon_0=0 +proj=laea +lat_0=-90', you could provide ['a=6371228.0', 'units=m', 'lon_0=0', 'proj=laea', 'lat_0=-90']. Since I didn't make the get_area_def function, I cannot speak as to why it is this way. I could add an example to show this functionality though!

Reply:
I almost think we should remove the list of proj4 arguments in the documentation. This is probably only supported because of the legacy format and how it formatted things. @mraspaud may have other thoughts.